lib/simple_forms_api_submission/s3.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# S3 Module for simple form submission
+# Return
+#   { success: Boolean, [error_message: String] }
+module SimpleFormsApiSubmission
+  class S3
+    attr_reader :region, :access_key_id, :secret_access_key, :bucket_name
+
+    def initialize(region:, access_key_id:, secret_access_key:, bucket_name:)
+      @region = region
+      @access_key_id = access_key_id
+      @secret_access_key = secret_access_key
+      @bucket_name = bucket_name
+    end
+
+    def upload_file(key, file)
+      obj = resource.bucket(bucket_name).object(key)
+      obj.upload_file(file)
+
+      { success: true }
+    rescue => e
+      { success: false, error_message: "S3 Upload failure for #{file}: #{e.message}" }
```

Comment:
What does a failure message look like? Just wondering if there's any chance for PII to leak in the error message.